cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -77,6 +77,26 @@
     {% set show_categories = cat_controls.show_preview_categories if cat_controls is defined else true %}
     <article class="o-post-preview">
         <div class="m-meta-header">
+            {% if post.categories.exists() and show_categories %}
+            <div class="m-meta-header_left">
+                {# Newsroom Blog category logic #}
+                {% import 'macros/category-slug.html' as category_slug %}
+                {% if cat_controls and 'newsroom' in cat_controls.page_type and is_blog(post) %}
+                    {{ category_slug.render(category='blog', href=url) }}
+                {% else %}
+                    {% for cat in post.categories.all() %}
+                        {% if loop.index > 1 %}
+                            <span class="m-meta-header_separator">|</span>
+                        {% endif %}
+                        {{ category_slug.render(category=cat.name, href=url) }}
+                    {% endfor %}
+                {% endif %}
+                {# pipe ("|") separator logic #}
+                {% if show_date %}
```

Comment:
Would it make sense to move this into the if block above it and have `if … elif … else`? Or does that combine two unrelated things?

Reply:
Ooo I guess that wouldn't work if you needed this and the stuff in the else block, never mind.

Reply:
They're unrelated.

The code above is for separating in the case of multiple categories. The `if show_date` is to prevent a Pipe character from rendering if categories are visible but date is hidden.

Reply:
👍